src/DataTableColumn.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/*
+ * =============================================================================
+ * This file is part of hguenot/base-app project.
+ * =============================================================================
+ * Copyright 2018 - Herve Guenot - https://github.com/hguenot
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ * =============================================================================
+ * 
+ * Date: 25/11/2018
+ * Time: 07:37

```

Comment:
@hguenot I guess you could just add `@author` and `@license` doc tags, it will be enough